python/lsst/pipe/base/__init__.py

```diff
@@ -3,19 +3,8 @@
 from ._instrument import *
 from ._status import *
 from ._task_metadata import *
-
-try:
-    from .argumentParser import *
-except ImportError:
-    # Gen2 imports are optional.
-    pass
 from .butlerQuantumContext import *
-
-try:
-    from .cmdLineTask import *
-except ImportError:
-    # Gen2 imports are optional.
-    pass
+from .cmdLineTask import *  # Remove once profile can be deleted.
```

Comment:
This does not import anything as `__all__` is now empty in that module?

Reply:
Hmm. If profile was never exported I should remove this line completely.